### PR TITLE
Documentation to Stateless and ES6 components (tooling integration)

### PR DESCRIPTION
**Pull request by ankitml**
Reference Issue : https://github.com/facebook/react/issues/3710
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`grunt test`).
5. Make sure your code lints (`grunt lint`) - we've done our best to make sure these rules match our internal linting guidelines.
6. If you haven't already, complete the [CLA](https://code.facebook.com/cla).

### CODE REVIEW

File: docs/docs/09.1-language-tooling.md [hunk 1]
```diff
@@ -35,30 +35,23 @@ By default JSX files with a `.js` extension are transformed. Run `babel --help`
...
 
-```
+```js
 $ cat test.js
-var HelloMessage = React.createClass({
-  render: function() {
-    return <div>Hello {this.props.name}</div>;
-  }
-});
+const HelloMessage = (name) => {
```
Comment (zpao) [inline docs/docs/09.1-language-tooling.md]: This should still take a props object, not `name` as a standalone argument.

Comment (ankitml) [inline docs/docs/09.1-language-tooling.md]: Ok. Will update in next commit.

**Comment by ankitml** [comment]
@zpao I have updated the Pull request with proper props as component arguments.

**Comment by facebook-github-bot-0** [comment]
@ankitml updated the pull request.

**Comment by gaearon** [comment]
Thanks! I tweaked and cherry-picked it as 8897a881f5130c3652345b3063587bd24496024e, leaving the commit attributed to you.

**Pull request closed without merging**